litgpt/utils.py

```diff
@@ -256,7 +256,7 @@ def _write_storage_and_return_key(self, storage):
         if storage.device.type != "cpu":
             storage = storage.cpu()
         num_bytes = storage.nbytes()
-        self.zipfile.write_record(name, storage.data_ptr(), num_bytes)
```

Comment:
Unfortunately, this doesn't work on Torch 2.2.2.
And that is the latest possible version, that can be installed on MacOS. That's why there is an error in CI for the MacOS runner.